DuckDuckGo/Tab/UserScripts/DebugUserScript.swift

```diff
@@ -66,7 +67,7 @@ final class DebugUserScript: NSObject, StaticUserScript {
 
     private func handleLog(message: WKScriptMessage) {
         // Used to log JS debug events. This is noisy every time a new tab is opened, so it's commented out unless needed.
-        // os_log("%s", type: .debug, String(describing: message.body))
+        Logger.general.debug("Handle log \(String(describing: message.body))")
```

Comment:
if it is noisy, should we comment it out? or perhaps we should make a separate not general log for it so it can be easily filtered out?

Reply:
very true, commented for now